### DISCUSSION TITLE
i want to psd convert to jpg  use  magick, so i want to known  which lib i need to down

### DISCUSSION THREAD

**Discussion by xiaoxiao-1882** [ImageMagick/ImageMagick#7016] [Help]
### ImageMagick version

7.1.1.-26

### Operating system, version and so on

centos

### Description

i want to psd convert to jpg  use  magick, so i want to known  which lib i need to down

**Comment by snibgo** [comment]
Looking at coders/psd.c, the only library used for reading and writing PSD files is zlib.

Does that answer the question?
